.github/workflows/ci.yml

```
@@ -8,6 +8,10 @@ jobs:
 
     strategy:
       matrix:
+        xcode:
+          - "11" # Swift 5.1
+          - "11.4" # Swift 5.2
```

Would it make sense to test with the latest stable versions instead of the initial versions?

```suggestion
          - "11.3.1" # Swift 5.1
          - "11.7" # Swift 5.2
```

---

Indeed it would. Nice catch!